src/netaddress.cpp

```diff
@@ -182,7 +182,7 @@ bool CNetAddr::IsTor() const
 
 bool CNetAddr::IsLocal() const
 {
-    // IPv4 loopback
+   // IPv4 loopback (127.0.0.0/8 or 0.0.0.0/8)
```

Comment:
Indentation should always be 4 spaces

Reply:
Don't forget to install `clang-format` and run the https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy script to preempt whitespace nitpicking.